facebook/qsf-tools/qsf-differ.R

```diff
@@ -188,7 +190,10 @@ diff_question <- function(names, change_type=c("Choices", "QuestionText", "Displ
 #' @param reference_qsf named list of trimmed output from `get_qsf_file` for survey that
 #'   contains descriptive info about a particular type of change. For "removed"
 #'   questions, should be older survey, else newer survey.
-create_diff_df <- function(questions, change_type=c("Added", "Removed", "Choices", "QuestionText", "DisplayLogic", "Subquestions"), reference_qsf) {
+create_diff_df <- function(questions, change_type=c("Added", "Removed",
+                                                    "Choices", "QuestionText",
+                                                    "DisplayLogic", "Subquestions"),
+                           old_reference_qsf, new_reference_qsf) {
```

Comment:
I think the documentation block for this function will need to be updated with the new argument names